### PR TITLE
nk - 1st PR 

### PR DESCRIPTION
**Pull request by engineerklimov**
@nsofiia please, review my code :)

### CODE REVIEW

File: requirements.txt
Comment (andronic23a) [inline requirements.txt:1]: nice!

**Review by andronic23a — APPROVED** [review]

**Review by annasadovskaya — APPROVED** [review]
Looks good to me

**Review by nsofiia — COMMENTED** [review]
Looks great!

**Review by nsofiia — COMMENTED** [review]
Nicely done!

**Pull request merged**